lib/test.js

```diff
@@ -434,16 +434,15 @@ Test.prototype['throws'] = function (fn, expected, msg, extra) {
         fn();
     } catch (err) {
         caught = { error : err };
-        var message = err.message;
-        delete err.message;
-        err.message = message;
```

Comment:
The impact they have is, if an error had a getter as its "message" property, this converts it to a straight assignment.

However, if these are important, there should be a test that ensures they're not removed.